src/app/shared/navigation-focus/navigation-focus.ts

```diff
@@ -9,16 +11,32 @@ let lastTimeoutId = -1;
 export class NavigationFocus implements OnInit {
   @HostBinding('tabindex') role = '-1';
 
-  constructor(private el: ElementRef) {}
+  constructor(private el: ElementRef, private router: Router) {}
 
   ngOnInit() {
-    clearTimeout(lastTimeoutId);
-    // 100ms timeout is used to allow the page to settle before moving focus for screen readers.
-    lastTimeoutId = window.setTimeout(() => this.el.nativeElement.focus({preventScroll: true}),
-      100);
+    let previousUrl = window.location.hostname;
+    this.router.events
+      .pipe(filter((event: Event) => event instanceof NavigationEnd))
+      .subscribe(() => {
+        const afterUrl = window.location.hostname;
```

Comment:
It looks like this variable is only used once, might as well just inline it

Reply:
Changed to using `id` from `RouterEvent`